generators/servicehook/templates/ExtId.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It'll be nice to switch at some point to the new 2017 .csproj format. Soooo much nicer.